.github/workflows/build-images.yaml

```diff
@@ -69,6 +69,9 @@ jobs:
       - name: Run make bundle
         if: ${{ github.ref_name != env.MAIN_BRANCH_NAME }}
         run: make bundle REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }} IMAGE_TAG=${{ github.ref_name }}
+      - name: Run make bundle (main)
```

Comment:
I think that's the only thing left unaddressed… Am not sure where this whole `IMAGE_TAG=latest VERSION=0.0.0` thing is coming from or what it addresses, @didierofrivia Can you shed some light?

Reply:
I think I got this. 

The key is the conditional `if`. 

* `if: ${{ github.ref_name != env.MAIN_BRANCH_NAME }}` -> The bundle will reference kuadrant operator `github.ref_name`
* `if: ${{ github.ref_name == env.MAIN_BRANCH_NAME }}` -> The bundle will reference kuadrant operator `latest` (instead of `main`). BTW both `latest` and `main` are created and pushed to quay.

For me, specifying the `VERSION` here is useless, as the default value will be `0.0.0`. I think this is just to make sure `latest` bundle has always ''0.0.0` version, no matter what is specified in the Makefile. 

For the case of the bundle, the proposed changes are ok to me, although not very relevant. Thus, the bundle will reference `latest` with these changes and otherwise, the referenced image will be `main`.